phpBB/styles/prosilver/theme/content.css

```diff
@@ -383,8 +383,7 @@ dl.faq dt {
 }
 
 .content ul, .content ol {
-	margin-bottom: 1em;
-	margin-left: 3em;
+	margin: 1em 0 1.2em 3em;
```

Comment:
I didn't notice they were different from each other. Then I assume that code/list have different font sizes? In that case,  would it make sense to use px or--even better--**rem**? Apparently [rem is supported on everything but IE 7](http://caniuse.com/#feat=rem).

I don't care either way, `0.8em 0 0.9em 3em` is fine by me.